app/controllers/blogs_controller.rb

```diff
@@ -1,5 +1,12 @@
 class BlogsController < ApplicationController
-  def index; end
+  def index
+    @user = User.find(params[:user_id])
+    @blogs = @user.recent_posts
```

Comment:
- [ ] Here in the blog's index, you have to show all the blogs written by the users and not only the recent blog with the 5 recent comments.

Reply:
Thank you. It is fix